README.md

```diff
@@ -110,6 +110,44 @@ if Mix.env == :dev do
 end
 ```
 
+## Disabling ansi_enabled setting
+
+The config `:ansi_enabled` is defaulted to `true` which will force the 
+console to have [ANSI coloring](https://hexdocs.pm/elixir/IO.ANSI.html#enabled?).  
+
+If Elixir can detect during startup that both `stdout` and `stderr` are terminals,
```

Comment:
From the documentation, elixir will try to set it for you

<img width="876" alt="screen shot 2017-04-02 at 8 57 40 am" src="https://cloud.githubusercontent.com/assets/48086/24587325/81b57256-1782-11e7-98bc-a1e8cc3c58dc.png">